### ISSUE TITLE
“Pollen Time” streaming event: May 7, 2020

### ISSUE THREAD

**Issue by otherjoel** [mbutterick/pollen-users#45]
I’m planning to have the next “Pollen office hours” type event on May 7. I will edit this comment shortly with a link to all the information.

**Comment by otherjoel** [comment]
The mic check on Sunday “went well” by which I mean a few people were able to hear my words and watch my screen and said it worked fine.

We started a [new repo for the Pollen Cookbook](https://github.com/otherjoel/cookbook-pollen) project mentioned above.

**Comment by celtic-coder** [comment]
Hi Joel,

The [YouTube video](https://www.youtube.com/watch?v=_zTabyOJ99U) has been removed. Are there plans to re-upload at some stage in the future?

Kind Regards,
Liam

**Comment by otherjoel** [comment]
Yeah, I screwed that up; YouTube offered for me to “edit” the video as soon as the session ended. I removed a few minutes of dead air at the beginning, but the result was saved as an entirely new video with its own link. Seeing two, I deleted the unedited one, breaking the link.

Long story short, I have fixed the link in my original comment above. I'll send out another email today or tomorrow and mention the fixed link as well.

**Comment by celtic-coder** [comment]
Hi Joel,

Your original comment looks to be linked to the version that is unavailable. Is the updated version now at https://www.youtube.com/watch?v=VGlzbH63WUE ?

Kind Regards,
Liam

**Comment by otherjoel** [comment]
You're right! I edited the wrong link. Should be fixed now. Yes the one you posted is correct.

**Comment by celtic-coder** [comment]
Great! Glad that it is sorted! 😄
